doc/classes/Animation.xml

```diff
@@ -28,7 +28,7 @@
 			<argument index="1" name="at_position" type="int" default="-1">
 			</argument>
 			<description>
-				Add a track to the Animation. The track type must be specified as any of the values in the TYPE_* enumeration.
+				Adds a track to the Animation. The track type must be specified as any of the values in the[code]TYPE_*[/code] enumeration.
```

Comment:
I think you can replace "TYPE_*" with [enum TrackType] to get a hyperlink to that section of the docs. Also don't forget to restore the space before it!!

Reply:
I'm not sure if it's useful to specify in the description that some arguments must have a specific enum type, as it's also stated in the method signature (and the enum is clickable there).

By the way, @akien-mga told me he plans to replace all instances [code]TYPE_*[/code] with proper enums (I already did a small part of it).